### ISSUE TITLE
Add group for imported entries

### ISSUE THREAD

**Issue by uok** [beemdevelopment/Aegis#1491]
Thanks for this awesome app, it works really great! :+1:  
I'm thankful for the import feature which helps with migrating from Authy.

My small suggestion: please add group "import" or "imported from Authy" or similar for all imported entries.
Thanks!

**Comment by michaelschattgen** [comment]
While I initially liked the idea of having a separate group for all imported entries, I find myself questioning what the use case for this would be. Why would you want to have the entries in this group? Could you elaborate on that?

**Comment by uok** [comment]
It seems that an increasing number of websites/services (in some cases as an overkill) are using 2FA so the list of entries keeps growing. My idea was to separate the imported entries from existing ones like a "todo list" which I need to check if working correctly. Also if user is importing from more than one app it might be helpful to have this separation.

**Comment by michaelschattgen** [comment]
Makes sense! The more I'm thinking about this, the more I'm thinking about a "assign groups" feature during the import process where the user can easily add a group to all entries or to a preferred selection of entries. What the user calls the group is up to them, but in this way it covers a wider range of use cases than just the "Imported from ..." group. Would that be sufficient enough?

What do you think @alexbakker?
